BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -14,8 +14,23 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define RSA_KEY_VB_SIGNATURE     SIGNATURE_32 ('$', '_', 'V', 'B')
 #define RSA_KEY_IPP_SIGNATURE    SIGNATURE_32 ('$', 'I', 'P', 'P')
+#define PUBKEY_IDENTIFIER        SIGNATURE_32 ('P', 'U', 'B', 'K')
+#define SIGNATURE_IDENTIFIER     SIGNATURE_32 ('$', 'I', 'P', 'P')
```

Comment:
why do we use $IPP again.  It conflicts with RSA_KEY_IPP_SIGNATURE.

Reply:
let us just use  'SIGN'

Reply:
We have the changed  identifier in Signing tools. Let me update it here too.
Anyways we are not consuming this identifier as part of this patch.